src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) throws DukeException {
+        try {
+            if(description=="") {
```

Comment:
I think string comparison should be done with equals method. Do refer to the lecture notes for more details.